### ISSUE TITLE
Possibility for cluster

### ISSUE THREAD

**Issue by svennd** [arjitc/DCIMStack#48]
It would be nice to be able to add clusters. We have two small clusters one cluster has 4 nodes (2U) (each a CPU) and the other 8 nodes (4U) While I can work around this, but for blade servers (http://www.supermicro.nl/products/MicroBlade/) this would be more difficult. Also memory is not equally for all, head node has lesser.

A free field would maybe be easier to add, cause for example all these nodes have ipmi/mac aswell.

Just a suggestion really.

**Comment by arjitc** [comment]
Good idea, I didn't think about this before but after thinking about this now and I'm sure this will be required by many, added to my todo list :+1:
